### ISSUE TITLE
yarn upgrade-interactive 0.17.0. UI/UX

### ISSUE THREAD

**Issue by DmitryAuine** [yarnpkg/yarn#1869]
This is awesome, but it has lack of UI
When pkg.json is empty, there aren't any modules to upgrade. So I can't select any. There is no key for exit, for cases when I haven't any pgks to upgrade, or just want to exit this tool
Curently 0.17.0
```
Choose which packages to update. (Press <space> to select, <a> to toggle all, 
<i> to inverse selection)

>> You must choose at least one package.

```
So I stuck here. How I can select if there isn't any modules? :)

Expected: Pls add button to exit from this utility
Any platform

**Comment by torifat** [comment]
Reported in #1843 & Fixed in #1846

**Comment by Daniel15** [comment]
Closing as duplicate of #1843. Thank you for trying Yarn! 😄

**Comment by msegado** [comment]
Seems this issue was more about exit functionality than an empty listing ("Expected: Pls add button to exit from this utility"). For posterity, you can just use Ctrl+C to exit, though it might not hurt to make that explicit in the prompt.

**Comment by DmitryAuine** [comment]
@msegado Sure i can, but guys we talk about interactive tool, right?

**Comment by torifat** [comment]
@DmitryAuine feel free to send a PR. We would love to merge it.

You can do it like `npm-check` does. https://github.com/dylang/npm-check

<img width="454" alt="screenshot 2016-11-17 12 22 55" src="https://cloud.githubusercontent.com/assets/208544/20378479/9ff3cdf8-acc0-11e6-9346-35e148820589.png">

**Comment by dguo** [comment]
I looked into this because I also expected an instruction for how to quit, especially because I seem to need to Ctrl+C twice to actually quit. However, the instruction comes from [Inquirer.js](https://github.com/SBoudrias/Inquirer.js). See SBoudrias/Inquirer.js#454

I would actually expect pressing enter without selecting anything to quit for me, with maybe a warning that nothing was updated, rather than telling me that I have to select at least one package. Does that seem reasonable?
